### PR TITLE
Fourth homework for check ready

### PR DESCRIPTION
**Pull request by nvkaip**

### CODE REVIEW

File: FuelType.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.homework;
+
+public enum FuelType{
+    PATROL, DIESEL, LPG
+        }
```
Comment (nvkaip) [inline FuelType.java]: This is Builder pattern interpretation

File: SpamAnalyzer.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    @Override
+    public String[] getSpamKeywords() {
+        return spamKeywords;
+    }
+
+    @Override
+    protected Label getLabel() {
+        return Label.SPAM;
+    }
+}
```
Comment (nvkaip) [inline SpamAnalyzer.java]: Next is for https://stepik.org/lesson/12772/step/9?unit=3120

File: SquareRootExceptionApp.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        System.out.println(sqrt(0));
+    }
+
+    public static double sqrt(double x) {
+        if (x < 0) {
+            throw new IllegalArgumentException("Expected non-negative number, got " + x);
+        }
+        return Math.sqrt(x);
+    }
+}
```
Comment (nvkaip) [inline SquareRootExceptionApp.java]: Next is continue for Text Analyzer

File: CarShopApp.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.homework;
+
+public class CarShopApp {
+
+    public static void main(String[] args) {
+        Car car = new CarBuilder().setBrand("BMW").setModel("X3").setFuelType(FuelType.PATROL).getCar();
```
Comment (Sofasmile) [inline CarShopApp.java]: very long line
Car car = new CarBuilder()
.setBrand("BMW")
.setModel("X3")
.setFuelType(FuelType.PATROL)
.getCar();

File: FuelType.java [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package com.homework;
+
+public enum FuelType{
+    PATROL, DIESEL, LPG
```
Comment (Sofasmile) [inline FuelType.java]: each word in the separate lines

**Pull request merged**